.github/workflows/release.yml

```
@@ -1,6 +1,11 @@
 name: Release
 on:
   workflow_dispatch:
+      inputs:
+        already_published:
+          description: 'Has the release already been published to maven central?'
+          type: boolean
+          default: false
```

while i think it's still valid yaml, theres a couple extra spaces here

```suggestion
  workflow_dispatch:
    inputs:
      already_published:
        description: 'Has the release already been published to maven central?'
        type: boolean
        default: false
```